addons/l10n_fr_invoice_addr/views/report_invoice.xml

```diff
@@ -0,0 +1,81 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <template id="report_invoice_document" inherit_id="account.report_invoice_document">
+        <t t-set="o" t-value="o.with_context(lang=lang)"/>
+
+        <!--
+            ADDRESS BLOCK
+        -->
+
+        <xpath expr="(//t[@t-set='address']/address)[last()]" position="replace">
+            <t t-set="partner" t-if="o.l10n_fr_is_company_french()" t-value="o.partner_id.parent_id or o.partner_id"/>
+            <t t-set="partner" t-if="not o.l10n_fr_is_company_french()" t-value="o.partner_id"/>
+            <address t-field="partner.self" class="m-0"
+                     t-options='{"widget": "contact", "fields": ["address", "name"], "no_marker": True}'/>
+
+            <t t-if="o.l10n_fr_is_company_french() and partner.siret">
+                SIRET: <t t-esc="partner.siret"/>
+            </t>
+
+            <div class="mb-3"/>
+        </xpath>
+
+        <!--
+            INFORMATIONS BLOCK
+        -->
+
+        <!-- Shipping Address -->
+        <xpath expr="//div[@name='shipping_address_block']" position="attributes">
+            <attribute name="groups"/>
+            <attribute name="t-if">
+                o.l10n_fr_is_company_french() or o.env.user.has_group('sale.group_delivery_invoice_address')
+            </attribute>
+        </xpath>
+
+        <!-- Billing Address -->
+        <xpath expr="//div[@id='informations']" position="inside">
+            <t t-if="o.l10n_fr_is_company_french() and o.partner_id.parent_id">
+                <div class="col-auto col-3 mw-100 mb-2">
+                    <div class="font-weight-bold">Billing Address:</div>
+                    <p class="m-0">
```

Comment:
do we really need a p balise, i think that if we use a t or a span you don't need the class